docs/sources/mimir/configure/configure-otel-collector.md

```diff
@@ -107,6 +107,42 @@ service:
       exporters: [..., prometheusremotewrite]
 ```
 
+## Work with default OpenTelemetry labels
+
+OpenTelemetry metrics use resource attributes to describe the set of characteristics associated with a given resource, or entity producing telemetry data. For example, a host resource might have multiple attributes, including an ID, an image, and a type.
+
+To optimize the storage of and ability to query this data, Mimir automatically assigns the following OTel resource attributes as labels at the time of ingestion, with periods (`.`) replaced by underscores (`_`):
+
+- `service.instance.id`
+- `service.name`
+- `service.namespace`
+- `service.version`
+- `cloud.availability_zone`
+- `cloud region`
+- `container.name`
+- `deployment.environment.name`
+- `k8s.cluster.name`
+- `k8s.container.name`
+- `k8s.cronjob.name`
+- `k8s.daemonset.name`
+- `k8s.deployment.name`
+- `k8s.job.name`
+- `k8s.namespace.name`
+- `k8s.pod.name`
+- `k8s.replicaset.name`
+- `k8s.statefulset.name`
+
+{{< admonition type="note" >}}
+Some of these labels are mutually exclusive.
+{{< /admonition >}}
+
+To update this list, ...
```

Comment:
@aknuds1 and @jesusvazquez , could you please help me with this missing information? Thanks!

Reply:
Right now the only way is to reach out through support and let us know so that we can modify how to do it.

Reply:
Thanks, updated!